classes/03class/exercises/c03-docker02/starkmatt/c03-docker02.txt

```diff
@@ -0,0 +1,37 @@
+##Exercise 2###
+
+##Webserver 01 Build/Run
+➜  starkmatt git:(starkmatt/c03-docker02) ✗ docker build -t  exercise02:v1 -f Dockerfile.v1 . 
+➜  starkmatt git:(starkmatt/c03-docker02) ✗ docker run -dp 8081:80 exercise02:v1
+52851b9c34fe4814a8dcdd7202b382209fae8c5005ebe95ad5ef6b5ee54f3acb
+
+##Web Server 02 Build/Run
+➜  starkmatt git:(starkmatt/c03-docker02) ✗ docker build -t  exercise02:v2 -f Dockerfile.v2 . 
+➜  starkmatt git:(starkmatt/c03-docker02) ✗ docker run -dp 8082:80 exercise02:v2
```

Comment:
You're not mapping the required folder with this command.

Reply:
Hey Kiko,
Not to sure what you mean ?

Reply:
in the v2 image there is nothing in the htdocs folder, so you need to map the local folder when running the container:
```
Run a second container using the exercise02:v2 image exposing the container port(80) to your localhost port 8082. Additionaly, map the folder classes/03class/docker/artifacts/c03-docker02/ of this repo into the folder /usr/local/apache2/htdocs/ of the container during it's execution. You should be able to access the webserver throught http://localhost8082
```